contributed_definitions/nyaml/NXsample_substrate.yaml

```diff
@@ -0,0 +1,103 @@
+category: base
+doc: |
+  Any information on the sample substrate.
+type: group
+NXsample_substrate(NXobject):
+  material(NX_CHAR):
+    doc: |
+      Material of the substrate in direct contact with the sample.
+  state(NX_CHAR):
```

Comment:
You mean thermodynamic state "solid", "gaseous", then why not an enum for these classical controlled vocabulary item from classical constitution theory?

Reply:
We would have to ask @rettigl and @Tommaso-Pincelli since the concept of a sample state was introduced in the MPES framework. I don't think that it is meant to be "solid", "gaseous", but rather things like pellets, powders, foil, thin films, and so on. So basically, a duplication of "type" in NXsample. Not sure though.

Reply:
I'm not so sure either anymore, Tom brough this in.

Reply:
This was removed for now, maybe there should be a new PR describing these concepts.